Improvements

coldcite opened the pull request:
- Proper enemy names. Added support for Complex Doom, LCA, RM, Ark, HEM,
HAF. Easily extendable through language lump variables.
- Changed floating damage font to BIGFONT sprites. (_Old sprites are still defined, not replaced_)
- Changed HP bar not showing on some special enemies with an
APROP_RenderStyle different to STYLE_Normal (_**This needs checking**: I just started learning how this stuff works and I don't really know if it was set like that for some specific reason_) and also on invulnerable
foes. 
- Added INVUL (yellow) HP bar.

coldcite committed:
Improvements

- Proper enemy names. Added support for Complex Doom, LCA, RM, Ark, HEM,
HAF. Easily extendable through language lump variables.
- Changed floating damage font to BIGFONT sprites.
- Changed HP bar not showing on some special enemies with an
APROP_RenderStyle different to STYLE_Normal and also on invulnerable
foes.
- Added INVUL (yellow) HP bar.

j-palomo commented:
I like these changes. Unfortunately, I am unable to merge it until I get permission to use code from the HPBar & HP Tracker Mod by PkmX.

coldcite commented:
Of course.
The specific chunk of code borrowed is the enemy definition list, used as a base mostly, as it required some extensive editing.

coldcite committed:
Enemy names update (METADOOM)

Added  proper enemy names for METADOOM mod.

coldcite committed:
Enemy names update (METADOOM)

Dumb fix.

coldcite committed:
Enemy names update (Smooth Doom, Beautiful Doom)

Added proper enemy names for Smooth Doom mod.
Added proper enemy names for Beatuful Doom mod.
Tweaked enemy names for MetaDoom mod.

j-palomo commented:
I've lost interest in this project in general. You may continue work on it as a separate project if you want. It would probably be a good idea to rename it to avoid confusion. I was unable to contact PkmX. I sent an email, but never got a reply.
